variables.tf

```diff
@@ -45,6 +45,12 @@ variable "ignore_public_acls" {
   description = "Whether Amazon S3 should ignore public ACLs for this bucket"
 }
 
+variable "object_ownership_type" {
```

Comment:
We order the variables in this file alphabetically. Can you put this var on the right place?

Reply:
Organized alphabetically in the new version